lib/solidus_stripe/seeds.rb

```diff
@@ -0,0 +1,21 @@
+# frozen-string-literal: true
+
+module SolidusStripe
+  # Seed data to seamlessly integrate Solidus with Stripe.
+  module Seeds
+    # Default name for the `Spree::RefundReason` used for Stripe refunds.
+    #
+    # Changed via {SolidusStripe::Configuration#refund_reason_name}.
+    #
+    # @return [String]
+    DEFAULT_STRIPE_REFUND_REASON_NAME = "Refund generated from Stripe"
+
+    def self.refund_reasons
+      Spree::RefundReason.find_or_initialize_by(
+        name: DEFAULT_STRIPE_REFUND_REASON_NAME
+      ).tap do
+        _1.update!(mutable: false) unless _1.persisted?
+      end
```

Comment:
I think here we can use `find_or_create_by` with a block, which is there precisely for the extra creation attributes:

```rb
Spree::RefundReason.find_or_initialize_by(name: DEFAULT_STRIPE_REFUND_REASON_NAME) do
  _1.mutable = false
end
```

https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-find_or_create_by

Reply:
Didn't know that :+1: Changed.